server/methods/messageSearch.js

```diff
@@ -209,11 +209,11 @@ Meteor.methods({
 
 		if (Object.keys(query).length > 0) {
 			query.t = {
-				$ne: 'rm', // hide removed messages (useful when searching for user messages)
-			};
-			query._hidden = {
-				$ne: true, // don't return _hidden messages
+				$in: ['msg'], // only the ones we believe should be searched for
```

Comment:
Doesn't need to be a `$in`

Reply:
This is because we had more than one type, like the system messages